components/ReferenceManager/references/reference_table/ReferencesTable.tsx

```diff
@@ -71,6 +95,82 @@ export default function ReferencesTable({
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [pdfIsOpen, setPDFIsOpen] = useState<boolean>(false);
   const [pdfUrl, setPdfUrl] = useState<string>("");
+  const [dragStarted, setDragStarted] = useState(false);
+  const [rowDraggedOver, setRowDraggedOver] = useState();
+  const [rowDragged, setRowDragged] = useState();
+  const { currentOrg } = useOrgs();
+
+  const router = useRouter();
+  const apiRef = useGridApiRef();
+
+  const moveCitationToFolder = ({ moveToFolderId }) => {
+    const newReferenceData = referenceTableRowData.filter((data) => {
+      return data.id !== rowDragged;
+    });
+    setReferenceTableRowData(newReferenceData);
+    updateReferenceCitation({
+      payload: {
+        citation_id: rowDragged,
+        // TODO: calvinhlee - create utily functions to format these
+        project: parseInt(moveToFolderId, 10),
+      },
+      onSuccess: () => {
+        setRowDraggedOver();
+      },
+      onError: () => {},
```

Comment:
Can we add a `captureEvent` call in here? I think these call will be helpful insights.

```
captureEvent({ msg: "Failed to move folder", error, data: {
  rowDragged,
  moveToFolderId,
  data,
}})
```